portfolio/src/main/webapp/script.js

```diff
@@ -32,7 +36,7 @@ window.onload = function () {
 // @param letterDelay - duration in milliseconds of delay between letter keypresses. Must be > 0. 
 // @param wordDelay - duration in milliseconds of delay between presenting a finished word and starting the next one. Must be >0
 function startTypewriter(textSelector, words, letterDelay, wordDelay) {
```

Comment:
Maybe `letterDelayMs` or something similar that makes the unit explicit

Reply:
Thanks for pointing this out - I thought I fixed this previously but I don't think I did in the function signature. Fixed